stable/datadog/templates/system-probe-configmap.yaml

```diff
@@ -18,24 +13,19 @@ metadata:
 data:
   system-probe.yaml: |
     system_probe_config:
-      enabled: {{ $.Values.systemProbe.enabled }}
-      debug_port:  {{ $.Values.systemProbe.debugPort }}
+      enabled: {{ $.Values.datadog.systemProbe.enabled }}
+      debug_port:  {{ $.Values.datadog.systemProbe.debugPort }}
       sysprobe_socket: /opt/datadog-agent/run/sysprobe.sock
-      enable_conntrack : {{ $.Values.systemProbe.enableConntrack }}
-      bpf_debug: {{ $.Values.systemProbe.bpfDebug }}
-{{- if eq .Values.systemProbe.seccomp "localhost/system-probe" }}
+{{- if eq .Values.datadog.systemProbe.seccomp "localhost/system-probe" }}
```

Comment:
why did the `if` statement move up two lines?

Reply:
I think it is a merge issue. fixing it